cmd/admin-handlers.go

```diff
@@ -1263,10 +1263,6 @@ func (a adminAPIHandlers) ObjectSpeedTestHandler(w http.ResponseWriter, r *http.
 		}
 	}
 
-	defer objectAPI.DeleteObject(ctx, customBucket, speedTest+SlashSeparator, ObjectOptions{
-		DeletePrefix: true,
-	})
-
```

Comment:
this is on purpose we shouldn't retain temporary buckets @vadmeste

Reply:
> this is on purpose we shouldn't retain temporary buckets @vadmeste

By the way, any idea why would someone choose a custom bucket ? the PR here just avoids to remove speed test object when the user a bucket which already exists, which is not a common use case. Nothing will change with a regular 'mc support perf <alias>'

This is just an easy to generate random files, `mc support perf <alias> --bucket mybucket`

Reply:
by the way, there is no bucket removal, this is only removing a speedtest prefix in a bucket